WordPress/gutenberg
# Post author name: Add "post" prefix to CSS class

## Description

Updates a CSS class that was missed when the block changed name from "author name" to "post author name".

## Testing Instructions


1. Open a Post or Page.
2. Insert a post author name block
3. Enable the link option in the block settings sidebar
4. View the front. View source. Confirm that the class name on the link is "wp-block-post-author-name__link"

## Screenshots 

## Types of changes
Bug fix

## Checklist:
